examples/pytorch/mnist-ddp/mnist.py

```diff
@@ -0,0 +1,344 @@
+import argparse
```

Comment:
Since we don't yet support bypass the Python files to the TrainJobs, I would suggest that we keep the Python function in the Jupyter Notebook: https://github.com/kubeflow/training-operator/issues/2347
cc @shravan-achar